kafka_consumer/ci/kafka_consumer.rake

```diff
@@ -0,0 +1,65 @@
+require 'ci/common'
+
+def kafka_consumer_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+namespace :ci do
+  namespace :kafka_consumer do |flavor|
+    task before_install: ['ci:common:before_install']
+
+    task install: ['ci:common:install'] do
+      use_venv = in_venv
+      install_requirements('kafka_consumer/requirements.txt',
+                           "--cache-dir #{ENV['PIP_CACHE']}",
+                           "#{ENV['VOLATILE_DIR']}/ci.log", use_venv)
+      sh %(docker-compose -f #{ENV['TRAVIS_BUILD_DIR']}/kafka_consumer/ci/resources/docker-compose-single-broker.yml up -d)
+      Wait.for 2181
+      Wait.for 9092
+      sleep_for 5
+      sh %(docker run -d --name kafka_producer -v /var/run/docker.sock:/var/run/docker.sock -e HOST_IP=172.17.0.1 -e ZK=172.17.0.1:2181 -i -t wurstmeister/kafka /bin/bash -c '$KAFKA_HOME/bin/kafka-console-producer.sh --topic=test --broker-list=`broker-list.sh` < /etc/group')
+      sh %(docker run -d --name kafka_consumer -v /var/run/docker.sock:/var/run/docker.sock -e HOST_IP=172.17.0.1 -e ZK=172.17.0.1:2181 -i -t wurstmeister/kafka /bin/bash -c '$KAFKA_HOME/bin/kafka-console-consumer.sh --topic=test --zookeeper=$ZK --consumer-property group.id=my_consumer ')
```

Comment:
We're inevitably going to have some very long line in these, there's way to split them up in ruby or we can do some various interpolations. We should decide what we wanna do as a standard.